packages/ui/src/2_molecules/HelperButton/HelperButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import { Icon } from '../../1_atoms';
+import { Tooltip } from '../Tooltip';
+import styles from './HelperButton.module.css';
+
+const iconSize = 8.33;
+
+type HelperButtonProps = {
+  content: ReactNode;
+  className?: string;
+  dataAttribute?: string;
+};
+
+export const HelperButton: React.FC<HelperButtonProps> = ({
+  content,
+  className,
+  dataAttribute,
+}) => (
+  <Tooltip
+    content={content}
+    className={classNames(styles.wrapper, className)}
+    dataLayoutId={dataAttribute}
+  >
+    <div>
+      <Icon size={iconSize} icon="info" />
```

Comment:
```suggestion
      <Icon size={12} icon="info" />
```

We should set container size here so that SVG is sized correctly (currently it's 2px smaller than it should be). Already checked this with Gilad, he will update size in figma